src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,55 @@
+import classNames from 'classnames';
+import { FilterType } from '../../helpers/filterTodos';
+
+interface Props {
+  activeTodosCount: number;
+  hasCompletedTodo: boolean;
+
+  removeAllCompleted: () => void;
+  filterType: FilterType;
+  setFilter: React.Dispatch<React.SetStateAction<FilterType>>;
+}
+
+export const Footer: React.FC<Props> = ({
+  activeTodosCount,
+  hasCompletedTodo,
+
+  removeAllCompleted: deleteAllCompleted,
```

Comment:
Consider using the original prop name `removeAllCompleted` instead of renaming it to `deleteAllCompleted`. Consistency in naming helps to maintain readability and avoid confusion.